google_guest_agent/snapshot_listener.go

```diff
@@ -85,8 +85,8 @@ func listenForSnapshotRequests(address string, requestChan chan<- *sspb.GuestMes
 		logger.Infof("Attempting to connect to snapshot service at %s.", address)
 		conn, err := grpc.Dial(address, grpc.WithInsecure())
 		if err != nil {
-			logger.Errorf("Failed to connect: %v.", err)
-			continue
+			logger.Errorf("Failed to connect to snapshot service: %v.", err)
+			break
```

Comment:
altho it works bc the function is entirely one for loop, i think the intention here is maybe better captured by 'return'?

Reply:
Fixed (in both places)!